python/cuspatial/cuspatial/tests/binpreds/binpred_test_dispatch.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+import pytest
+from shapely.geometry import LineString, MultiPoint, Point, Polygon
+
+import cuspatial
+
+"""Test Dispatch"""
+
+"""Testing all combinations of possible geometry types and
+binary predicates is a complex task. This file contains the
+basic fixtures for all geometry types that are required to
+cover different possible test outcomes. This file also contains
+the dispatching system that uses each fixture to generate
+a test for each possible combination of geometry types and
+binary predicates.
+"""
+
+"""The following fixtures are used to generate tests for
+each possible combination of geometry types and binary
+predicates. The fixtures are combined in the test function
+in `test_binpreds_test_dispatch.py` to make the following
+tuple: (predicate, geotype, geotype, expected_result). The
+geotype fixtures are used to generate the first two
+elements of the tuple. The predicate fixture is used to
+generate the third element of the tuple. The expected_result
+fixture is used to generate the fourth element of the tuple.
```

Comment:
The actual function signature is a different, it has only 3 arguments.

```suggestion
tuple: (predicate, geotype_tuple, expected_result). The
geotype fixtures are used to generate the second 
elements of the tuple. The predicate fixture is used to
generate the first element of the tuple. The expected_result
fixture is used to generate the third element of the tuple.
```